### ISSUE TITLE
Migrate seedv2 to helm and upgrade to latest approved version

### ISSUE THREAD

**Issue by macintoshpie** [SEED-platform/seed#2432]
### Instance Information

Server Instance: `dev1, dev2, production, staging, etc`
Server SHA: `abcd123`

**Comment by nllong** [comment]
Should be 2.7.4 -- right?

**Comment by RDmitchell** [comment]
@macintoshpie / @nllong -- can you tell me when you are going to do this migration, so that I can give the users a heads up about when it will be happening?

**Comment by nllong** [comment]
We will set this up in parallel—That is having the existing seedv2 running along side the new helm version. (And done similar for seeddemostaging.)

Da: RDmitchell <notifications@github.com>
Data: martedì, 10 novembre 2020 12:04
A: SEED-platform/seed <seed@noreply.github.com>
Cc: Long, Nicholas <Nicholas.Long@nrel.gov>, Mention <mention@noreply.github.com>
Oggetto: Re: [SEED-platform/seed] Migrate seedv2 to helm and upgrade to 2.7.3 (#2432)

@macintoshpie<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmacintoshpie&data=04%7C01%7Cnicholas.long%40nrel.gov%7Cd3121bb7bd204115f96308d885ab77af%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637406318790473054%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BotITLIHI7%2Bfuu%2B4O6BSngS03j5NlvvcbQ%2FLLL0ngbc%3D&reserved=0> / @nllong<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnllong&data=04%7C01%7Cnicholas.long%40nrel.gov%7Cd3121bb7bd204115f96308d885ab77af%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637406318790473054%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8GfvgReOkvc4InuNV2%2B8OAV%2FfcWNQJ%2BqrKIqip%2Fp%2F%2Fo%3D&reserved=0> -- can you tell me when you are going to do this migration, so that I can give the users a heads up about when it will be happening?

**Comment by adrian-lara** [comment]
@aviveiros11 what's the latest here? Is seedv2 on helm? If not, where can she test this deployment?

**Comment by aviveiros11** [comment]
@adrian-lara @RDmitchell the seedv2 helm instance can be found here: 

ac672f5f2110845c0b1ebb35a5475d10-1718251163.us-east-1.elb.amazonaws.com 

and behaviour should be equivalent to that of production.

**Comment by RDmitchell** [comment]
@aviveiros11 

is this supposed to be production before the update to the new release? 

Not sure I understand the functionality of this ... ?

**Comment by RDmitchell** [comment]
@aviveiros11 

I guess this is before being updated with the new release, as this is the SHA

![image](https://user-images.githubusercontent.com/6314950/109882339-91b79700-7c2e-11eb-875a-b52cc1759248.png)

**Comment by RDmitchell** [comment]
Just let me know if I need to do anything for this. Thx !!

**Comment by aviveiros11** [comment]
Hey, @RDmitchell I should've provided more context here I apologize.  We are gradually moving our deployments to a Kubernetes/helm container orchestration system versus the traditional docker-compose that existed previously.  That endpoint that I shared above points to a mock production server environment to test that the container orchestration is behaving exactly like the production server.  This is just a preventative measure to ensure a seamless transfer when we actually migrate.  We were hoping you could quickly just poke around on this endpoint and run some general workflow tests to confirm the mirrored behaviour with production.  When we actually migrate fully to Kubernetes/helm this server will be on the latest version tagged in GH ~ end of march.  Let me know if you have any questions!

**Comment by adrian-lara** [comment]
@RDmitchell Would you be able to test SEEDs core features to make sure the instance is working as expected?

**Comment by RDmitchell** [comment]
@adrian-lara -- and this is on dev1 ? I can test this week

**Comment by aviveiros11** [comment]
@RDmitchell you can hit the K8's deployment by putting `ac672f5f2110845c0b1ebb35a5475d10-1718251163.us-east-1.elb.amazonaws.com` in your browser

**Comment by RDmitchell** [comment]
@aviveiros11 -- right, sorry, that was up in the beginning of the issue. I will do it this week.

**Comment by macintoshpie** [comment]
@RDmitchell were you able to test the new seed deployment?

**Comment by RDmitchell** [comment]
@macintoshpie -- No not yet. 

We shouldn't move the latest version to seedv2 until we have put it on staging and let the SEED users test it -- there are a lot of changes from the current version and I want to make sure everyone understands them. I will probably arrange to have webinars with the users to talk about all the new features. 

So we probably don't want to update seedv2 for at least a month -- I'm thinking late October.

**Comment by nllong** [comment]
Seedv2 now uses helm. We are not on the latest, but that upgrade should be a separate issue (but I don’t think we want an issue for it right now)
